src/Interpreters/Streaming/Aggregator.cpp

```diff
@@ -67,11 +67,8 @@ inline bool worthConvertToTwoLevel(
     size_t group_by_two_level_threshold, size_t result_size, size_t group_by_two_level_threshold_bytes, auto result_size_bytes)
```

Comment:
Can we fix the `auto result_size_bytes` to make it more explicit regarding its type